pkg/manifest/v1/alerts.go

```diff
@@ -26,7 +26,7 @@ type AlertCondition struct {
 // different from the AlertCondition type because it does not have an APIVersion.
 type AlertConditionInline struct {
 	Kind     string             `yaml:"kind" validate:"required"`
-	Metadata Metadata           `yaml:"metadata" validate:"required"`
+	Metadata Metadata           `yaml:"metadata,omitempty" validate:"omitempty"`
```

Comment:
is this intentional change? If so, please revert this, `metadata` is a required field for inline definitions

Reply:
My bad. I was experimenting with another change. Will revert now.